cosmos/providers/dbt/render.py

```diff
@@ -196,79 +197,78 @@ def render_project(
             logger.error("Unknown dbt type.")
             continue
 
+        # Add model to list models that will render.
+        models_to_render[model_name] = model
+
         # if test_behavior isn't "after_each", we can just add the task to the
         # base group and do nothing else for now
-        if test_behavior != "after_each":
+        if test_behavior != "after_each" or model.type != DbtModelType.DBT_MODEL:
```

Comment:
Type != DbtModelType.DBT_MODEL have no test.

Reply:
There's lots of room to improve the coverage! Any contributions in this sense would be very welcome